osu.Game.Rulesets.Osu/Objects/Drawables/DrawableOsuBlinds.cs

```diff
@@ -0,0 +1,298 @@
+// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using OpenTK.Graphics;
+using osu.Framework.Allocation;
+using osu.Game.Skinning;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.Textures;
+using osu.Game.Rulesets.Osu.Objects.Drawables.Pieces;
+using System;
+
+namespace osu.Game.Rulesets.Osu.Objects.Drawables
+{
+    /// <summary>
+    /// Element for the Blinds mod drawing 2 black boxes covering the whole screen which resize inside a restricted area with some leniency.
+    /// </summary>
+    public class DrawableOsuBlinds : Container
+    {
+        /// <summary>
+        /// Black background boxes behind blind panel textures.
+        /// </summary>
+        private Box box1, box2;
```

Comment:
Please rename to leftBox and rightBox. Maybe leftBlindBackground and rightBlindBackground would be even better.